### DISCUSSION TITLE
React app slow down after some time

### DISCUSSION THREAD

**Discussion by l0ttek** [vercel/next.js#17760] [Help]
We have a problem in our team with nextjs react app. We created an application, deploy it with custom server in azure app service and everything looks fine. But when some time passes app looks like slow down more and more. When it 's fresh (after restart azure app service), we have load time ~5s, but when some time passes (around 2-3 hour), it is 20s, 30s, 40s or more.

We have also very slow first contentful paint, it's around 5 second, but when we click on page on some links, it's few ms.

Maybe you have some ideas, what are we doing wrong? What we can impove? Maybe you encountered with similar problem?

We use:
react 16.13.1
next 9.5.2

Our package.json script used on production:
        "start": "set NODE_ENV=production && node server.js",
        "build": "next build",

**Comment by l0ttek** [comment]
Anyone met similar problem?

**Comment by JonCognioDigital** [comment]
We just put a new site live today and it's really slow on the production Azure app service.

If we use a different slot (call it something like -staging) it works lightning fast when one or two people are clicking but on the live site there's only 5-10 concurrent users and it can take over 10 seconds for a page to load.

Does Next use high memory in production? Is there something blocking when doing async requests in getServerSideprops?

We're unable to use static props unfortunately because we're getting data on every page from a CMS which changes after build time but it really shouldn't be this slow with such a small number of users, should it?

**Reply by VladymyrPylypchatin** [reply]
Same thing

**Reply by fabicon2882** [reply]
hi.. we have de same problem did you get some solution?... thanks a lot..

**Reply by VladymyrPylypchatin** [reply]
We solved it. It turned out to be that Next.js error page had a memory leak. Solved by updating to newer next js version
